Automattic/wp-calypso
# Shopping Cart: Remove more unused cart properties

## Proposed Changes

This is a follow-up to https://github.com/Automattic/wp-calypso/pull/86076 which removed unused shopping cart properties. This removes some additional ones which were missed the first time around.

## Testing Instructions

Verify that these properties are not used anywhere (apart from the `domains-mini-cart` https://github.com/Automattic/wp-calypso/pull/86199 which will remove those references).
